server.js

```diff
@@ -32,6 +49,38 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/directors', async (req, res) => {
+  const directors = await Director.find()
+  res.json(directors)
+})
```

Comment:
I'm still on the "shows" track – so if you wanted to list all "shows" (we'll figure out how to list the directors in the next endpoint), a base endpoint could look like this:

```
// ENDPOINT 1: List all shows
// Example: http://localhost:8080/shows
app.get('/shows', async (req, res) => {
  const allShows = await Show.find(req.query)
  res.json(allShows)
})
```
So pretty much the way you did for the directors, but with `find(req.query)` instead of just `find()`. Well done! 🙂